bigbone-rx/src/main/kotlin/social/bigbone/rx/RxAccountMethods.kt

```diff
@@ -260,6 +299,45 @@ class RxAccountMethods(client: MastodonClient) {
         accountMethods.unmuteAccount(accountId).execute()
     }
 
+    /**
+     * Add the given account to the user’s featured profiles.
+     * (Featured profiles are currently shown on the user’s own public profile.)
+     *
+     * @param accountId The ID of the [Account] in the database.
+     *
+     * @see <a href="https://docs.joinmastodon.org/methods/accounts/#pin">Mastodon API documentation: methods/accounts/#pin</a>
+     */
+    fun featureAccountOnProfile(accountId: String): Single<Relationship> = Single.fromCallable {
+        accountMethods.featureAccountOnProfile(accountId).execute()
+    }
+
+    /**
+     * Remove the given account from the user’s featured profiles.
+     * This reverts [featureAccountOnProfile].
+     *
+     * @param accountId The ID of the [Account] in the database.
+     *
+     * @see <a href="https://docs.joinmastodon.org/methods/accounts/#unpin">Mastodon API documentation: methods/accounts/#unpin</a>
+     */
+    fun unfeatureAccountFromProfile(accountId: String): Single<Relationship> = Single.fromCallable {
+        accountMethods.unfeatureAccountFromProfile(accountId).execute()
+    }
+
+    /**
+     * Sets a private note on a user.
+     *
+     * @param accountId The ID of the [Account] in the database.
+     * @param privateNote The comment to be set on that user. Provide empty string or null to clear currently set note.
+     *
+     * @see <a href="https://docs.joinmastodon.org/methods/accounts/#note">Mastodon API documentation: methods/accounts/#note</a>
+     */
+    fun setPrivateNotOnProfile(
```

Comment:
Typo: Should be `setPrivateNoteOnProfile`

Reply:
Whoopsie! ✍️

Reply:
✅